media/js/base/banners/firefox-app-store-banner.es6.js

```diff
@@ -13,6 +13,7 @@ function onLoad() {
 if (
     window.Mozilla.run &&
     window.site &&
+    !window.Mozilla.Client.isFirefox &&
     (window.site.platform === 'android' || window.site.platform === 'ios')
 ) {
     window.Mozilla.run(onLoad);
```

Comment:
So I'm only a bit careful here using Mozilla.Client without testing for it, so maybe check for it and use its values instead?

```suggestion
if (
    window.Mozilla.run &&
    window.Mozilla.Client &&
    window.Mozilla.Client.isMobile &&
    !window.Mozilla.Client.isFirefox
    //(window.site.platform === 'android' || window.site.platform === 'ios') // equiv?
) {
    window.Mozilla.run(onLoad);
```